src/test/java/SingUpTest.java

```diff
@@ -4,39 +4,41 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 public class SingUpTest {
+    public WebDriver driver;
 
-    @Test
-    public void negativeZipCodeTestLettersInsteadOfDigits() {
+    @BeforeMethod
```

Comment:
It's better to create driver once for all tests and then close it at the end.
Please use @Before/AfterClass annotations for that.